src/components/DrawerListItems.jsx

```diff
@@ -74,15 +100,20 @@ export function drawerListItems(currentPath, onNavigate, showAirdrop) {
         <ListItemText primary="Feegrant" secondary="coming soon" />
       </ListItemButton>
       <ListItemButton
-        disabled
+        disabled={nodeDataInfo ? false : true}
         onClick={() => onNavigate("/group")}
         sx={{ pb: 0.5, pt: 0.5 }}
         selected={currentPath === "/group"}
       >
         <ListItemIcon>
           <GroupsOutlinedIcon />
         </ListItemIcon>
-        <ListItemText primary="Groups" secondary="coming soon" />
+        <ListItemText primary="Groups"
+          secondary={
+            nodeDataInfo?.status === 'pending'? 'Checking sdk version...':
+            !nodeDataInfo ? 'Available on 46 version': 
```

Comment:
```suggestion
            nodeDataInfo?.status === 'pending'? 'Loading...':
            !nodeDataInfo ? 'Not supported': 
```

Reply:
It is better to show the user which version from it there. And instead of loading `checking skd version` making sense.
We will show it somewhere in our app. What version of the chain also?

Reply:
From the developer's point of view, it makes sense, but normal users don't know anything about the SDK.